src/utils/internal/jsonParse.test.ts

```diff
@@ -11,3 +11,7 @@ test('returns undefined without an error given an invalid JSON string', () => {
   expect(parse).not.toThrow()
   expect(parse()).toBeUndefined()
 })
+
+test('parse a single number as valid JSON string', () => {
```

Comment:
Nitpick: could you please move this test case to be the second in the file?
We usually structure our tests to list the success scenarios first and end up with exceptional or failure scenarios.